_redirects

```diff
@@ -10,3 +10,5 @@
 /fow-6				https://futureofwork.hs3.pl
 /calendar			/wydarzenia
 /spaceapi.json			https://spaceapi.in.hs3.pl/	307
+/gvt			https://www.meetup.com/pl-PL/hs3city/events/
```

Comment:
This points to all events, not just GVT. Is this fine?

Reply:
is OK